src/sentry/integrations/slack/utils.py

```diff
@@ -162,6 +162,9 @@ def get_channel_id_with_timeout(integration: Integration, name: str, timeout: in
                 items = client.get(
                     endpoint, headers=headers, params=dict(payload, cursor=cursor, limit=1000)
                 )
+            except ApiRateLimitedError as e:
+                logger.info("rule.slack.%s_list_rate_limited" % list_type, extra={"error": str(e)})
```

Comment:
We've recently started moving away from `"%s" % x` syntax in favor of `f"{x}"`.
```
logger.info(f"rule.slack.{list_type}_list_rate_limited", extra={"error": str(e)})
```

Reply:
Added, thanks!